# GPIO mask and state
uint8 MAX_INSTANCES = 8

uint64 timestamp			# time since system start (microseconds)
uint32 device_id			# Device id

uint32 state				# pin state mask
